packages/core/contracts/external/avm/interfaces/ArbSys.sol

```diff
@@ -0,0 +1,79 @@
+// Copied logic from https://github.com/makerdao/arbitrum-dai-bridge/blob/54a2109a97c5b1504824c6317d358e2d2733b5a3/contracts/arbitrum/ArbSys.sol
+// with changes only to the solidity version and comments.
+
+pragma solidity ^0.8.0;
+
+/**
+ * @notice Precompiled contract that exists in every Arbitrum chain at address(100),
+ * 0x0000000000000000000000000000000000000064. Exposes a variety of system-level functionality. ArbSys provides
+ * systems functionality useful to some Arbitrum contracts. Any contract running on an Arbitrum Chain can call the
+ * chain's ArbSys.
+ */
+interface ArbSys {
```

Comment:
is there no way to import these directly from the arbitrum package and therefore delete the `avm` directory in full? this is how we do things in the optimism contracts and it makes things clean af.

Reply:
I'm looking into if these are exported from the `arb-bridge-eth` package that arbitrum publishes. will report back shortly.

Reply:
damn. this package is private: https://github.com/OffchainLabs/arbitrum/blob/4b0f8d34b73036226d8a7455b5252d96e13bd7e4/packages/arb-bridge-eth/package.json?_pjax=%23js-repo-pjax-container%2C%20div%5Bitemtype%3D%22http%3A%2F%2Fschema.org%2FSoftwareSourceCode%22%5D%20main%2C%20%5Bdata-pjax-container%5D#L3